lib/chef/dist.rb

```diff
@@ -0,0 +1,21 @@
+class Chef
+  class Dist
+    # When referencing a product directly, like Chef (Now Chef Infra)
+    PRODUCT = "Chef Infra"
+
+    # The client's alias (chef-client)
+    CLIENT = "chef-client"
+
+    # the server tool's name (knife)
+    KNIFE = "knife"
+
+    # Name used for certain directories. Merge with chef_executable?
+    GENERIC = "chef"
+
+    # The chef executable, as in `chef gem install` or `chef generate cookbook`
+    EXEC = "chef"
```

Comment:
It seems like this one should be in DK/workstation where that tool actually exists.

Reply:
it's referenced in knife.rb. I'm adding them as I go